### PR TITLE
Simple exchange bug fixes

### PR DESCRIPTION
**Pull request by danishyasin33**

### CODE REVIEW

File: contract/src/auctionItems.js [hunk 1]
```diff
@@ -66,6 +66,25 @@ const start = (zcf) => {
     updater: availableItemsUpdater,
   } = makeNotifierKit();
 
+  const addToUserSaleHistory = (cardAmount) => {
+    try {
+      const validatedAmount = AmountMath.coerce(brands.Items, cardAmount);
```
Comment (danishyasin33) [inline contract/src/auctionItems.js:71]: What does coerce do?

File: ui/src/components/CardDisplay.jsx [hunk 2]
```diff
@@ -17,60 +17,103 @@ const CardDisplay = ({ handleClick, handleNFTMint }) => {
...
       if (userNftsMap[offer.id]) obj = { ...obj, ...userNftsMap[offer.id] };
       return obj;
     });
-    console.log('array:', arr);
     setMyCardLoader(false);
     return arr;
   };
   const getSecondaryCards = (params) => {
-    const ids = params?.userCards?.map((card) => card.id);
+    // const ids = params?.userCards?.map((card) => card.id);
```
Comment (danishyasin33) [inline ui/src/components/CardDisplay.jsx:75]: Remove commented out code

File: ui/src/services/swap.js [hunk 3]
```diff
@@ -12,47 +15,112 @@ const makeMatchingInvitation = async ({
...
   cardOffer,
   setLoading,
   onClose,
+  dispatch,
 }) => {
-  const result = await E(publicFacetSwap).makeMatchingInvitation({
+  console.log('cardPursePetname:', cardPurse.pursePetname);
+  console.log('cardbrand:', cardPurse.brand);
+  console.log(tokenPurses, 'tokenPurse');
+  console.log(cardDetail, 'cardDetail');
```
Comment (danishyasin33) [inline ui/src/services/swap.js:27]: Are these logs required somewhere? If not, remove please

File: contract/src/auctionItems.js (hunk 1 above)
Comment (Hussain-7) [reply contract/src/auctionItems.js:71]: It basically throws an error if the card Amount is not of the expected brand.And if it is of the right brand it returns validated amount

Comment (Hussain-7) [reply contract/src/auctionItems.js:71]: It basically throws an error if the card Amount is not of the expected brand.And if it is of the right brand it returns validated amount

File: ui/src/components/CardDisplay.jsx (hunk 2 above)
Comment (Hussain-7) [reply ui/src/components/CardDisplay.jsx:75]: Alright

File: ui/src/services/swap.js (hunk 3 above)
Comment (Hussain-7) [reply ui/src/services/swap.js:27]: No . Will remove

**Pull request merged**